.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: java
 jdk:
-- oraclejdk8
```

Comment:
Dropping oracle jdks as builds seems to be failing due to licencing
https://travis-ci.org/whiteclarkegroup/liquibase-linter/jobs/563893406